confluent_kafka/src/confluent_kafka.c

```diff
@@ -1508,52 +1581,38 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
 	conf = rd_kafka_conf_new();
 
         /*
-         * Default config (overridable by user)
+         * Set debug contexts first to capture all events including plugin loading
          */
+         if ((vo = PyDict_GetItemString(confdict, "debug"))) {
+                if (common_conf_set_special(confdict, conf, "debug", vo)) {
```

Comment:
no need for braces for single line blocks.

suggest you explicitly check for return value == -1, or change set_special to return 0 on error and 1 on success.

Reply:
Nope not needed, although arguably it does make it easier to read